components/data-components/contact-us-link.tsx

```diff
@@ -0,0 +1,69 @@
+export const ContactUsLink = ({ link }: { link: any }) => {
+  const getImgSize = () => {
+    switch (link.linkType) {
+      case 'tel':
+        return 'w-[20px] h-[20px]'
+      case 'address':
+        return 'w-[20px] h-[28px]'
+      case 'email':
+        return 'w-[27px] h-[20px]'
+
+      default:
+        break
+    }
+  }
+
+  const getLinkTxt = () => {
+    if (link.linkType === 'tel') {
+      return link.text.replace(/[0-9]/g, '')
+    } else {
+      return link.text
+    }
+  }
+
+  const getLinkVal = () => {
+    if (link.linkType === 'tel') {
+      return link.linkValue.replace(/\+/g, '') + '+'
+    } else {
+      return link.linkValue
+    }
+  }
+
+  const onLinkClick = () => {
+    if (link.linkType === 'email') {
+      //For Gmail (user has to be signed in to google)
+      window.location.href =
```

Comment:
this is not they way to do that.
Here is how an email link should be:

```
<a href="mailto:email@domain.com" className =grid grid-cols-auto-1fr gap-x-2">
    <Icon />
    <Text />
</a>
```


Also other links should be:

Phone:
```
<a href="tel:+123456789" className =grid grid-cols-auto-1fr gap-x-2">
    <Icon />
    <Text />
</a>
```

Address:
```
<a href="mailto:href="https://www.google.co.il/maps/search/מגדל הנביאים, ח׳ורי 2, קומה 6, חיפה"" className =grid grid-cols-auto-1fr gap-x-2">
    <Icon />
    <Text />
</a>
```